### ISSUE TITLE
coco dataset pretraining

### ISSUE THREAD

**Issue by mingminzhen** [zhanghang1989/PyTorch-Encoding#56]
Hi, it will be appreciated if you can provide the code about how to transform coco dataset into voc 2012 format.

**Comment by zhanghang1989** [comment]
Hi, it is just a label mapping. Then convert the mask
```
NUM_CHANNEL = 91
[] background
[5] airplane
[2] bicycle
[16] bird
[9] boat
[44] bottle
[6] bus
[3] car
[17] cat
[62] chair
[21] cow
[67] dining table
[18] dog
[19] horse
[4] motorcycle
[1] person
[64] potted plant
[20] sheep
[63] couch
[7] train
[72] tv
```

**Comment by zhanghang1989** [comment]
Hi, the COCO dataset is available now. Just didn't get a chance to train a model yet. You can start using it. https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/encoding/datasets/coco.py
